src/components/slider/slider.js

```diff
@@ -100,7 +103,12 @@ class Slider extends Component {
    * @memberof Slider
    */
   mouseEventToValue(e: any): number {
-    let offsetLeft = e.clientX - this._sliderElement.getBoundingClientRect().left;
+    let clientX;
+    if (e.touches && e.touches.length > 0) { clientX = e.touches[0].clientX }
+    else if (e.changedTouches) { clientX = e.changedTouches[0].pageX }
+    else { clientX = e.clientX; }
+
+    let offsetLeft = clientX - this._sliderElement.getBoundingClientRect().left;
```

Comment:
Isn't accessing `getBoundingClientRect` a performance hit?

Reply:
There's actually a better way to implement it with fewer calls to `getBoundingClientRect`. calculate the `getBoundingClientRect` only when player size is changed.
Will change the PR tomorrow

Reply:
10x, looking forward :-)